tests/rados/test_osd_onode_trimming.py

```diff
@@ -0,0 +1,149 @@
+"""
+Method to verify the onode trimming
+"""
+import datetime
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from tests.rados.monitor_configurations import MonConfigMethods
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_object = RadosOrchestrator(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_object)
+    ceph_nodes = kw.get("ceph_nodes")
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+    pool_name = "test-osdonode"
+    log.info("Running test case to verify trimming of onodes")
+
+    osd_list = []
+    onode_values = {}
+    log_line = [
+        "trim maximum skip pinned reached",
+    ]
+    mon_obj.set_config(section="osd", name="debug_bluestore", value="20/20")
+    try:
+        for node in ceph_nodes:
+            if node.role == "osd":
+                node_osds = rados_object.collect_osd_daemon_ids(node)
+                osd_list = osd_list + node_osds
+        # create pool with given config
+        log.info("Creating a replicated pool with default config")
+        assert rados_object.create_pool(pool_name=pool_name)
+
+        bench_cfg = {
```

Comment:
Do both bench read and bench write. This might increase the onode count faster.

Reply:
Included the bench write in the code.